Roles/Neutral/Jester.cs

```diff
@@ -38,6 +39,8 @@ public override void SetupCustomOption()
         SunnyboyChance = IntegerOptionItem.Create(Id + 7, "SunnyboyChance", new(0, 100, 5), 0, TabGroup.NeutralRoles, false)
             .SetParent(CustomRoleSpawnChances[CustomRoles.Jester])
             .SetValueFormat(OptionFormat.Percent);
+        RevealJesterUponEjection = BooleanOptionItem.Create(Id + 8, GeneralOption.RevealUponEject, false, TabGroup.NeutralRoles, true)
```

Comment:
Also `GeneralOption.RevealUponEject` not exit in code
So need change to `"Jester_"RevealUponEject"`
And add string for translation in [en_US.json](https://github.com/EnhancedNetwork/TownofHost-Enhanced/blob/main/Resources/Lang/en_US.json)

Reply:
Wait, what? I've been using `RevealJesterUponEjection` so how come it needs to be changed to `"Jester_"RevealUponEject"`? I understand the other comments(sorta lol) but this one throws me off